defopt.py

```diff
@@ -118,8 +118,24 @@ def _create_parser(funcs, *args, **kwargs):
     return parser
 
 
-class _Formatter(RawTextHelpFormatter, ArgumentDefaultsHelpFormatter):
-    pass
+class _Formatter(RawTextHelpFormatter):
+
+    # Modified from ArgumentDefaultsHelpFormatter to add type information,
+    # and remove defaults for varargs (which use _AppendAction instead of
+    # _StoreAction).
+    def _get_help_string(self, action):
+        info = []
+        if action.type is not None and '%(type)' not in action.help:
+            info.append('type: %(type)s')
+        if (type(action).__name__ is '_StoreAction'
```

Comment:
Is there a particular reason you're comparing the name rather than `isinstance(action, argparse._StoreAction)`?